spec/support/equipment_model_helper.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+module EquipmentModelHelper
```

Comment:
Yup. I think I'm going to rename them to 'mocker' instead of 'helper' for clarity. Also, I'd really prefer to make extensions of rspec spies in classes than these modules, but I'm having trouble getting rspec-mocks in classes in `spec/support`